client/src/components/footer/footer.js

```diff
@@ -224,10 +229,11 @@ class Footer extends Component {
                         : "block",
                   }}
                 >
-                  Next -{" "}
+                  Next
+                  {/* Next -{" "}
```

Comment:
You can delete unused code, we can always look back at older commits if we need to get this code back